### ISSUE TITLE
main.func_group failure

### ISSUE THREAD

**Issue by midenok** [tempesta-tech/mariadb#182]
### Reproduce
```sql
create or replace table t1 (grp int, a bigint unsigned);
insert into t1 values (null,null);
select grp,min(a),max(a) from t1 group by grp;
+------+--------+--------+
| grp  | min(a) | max(a) |
+------+--------+--------+
| NULL |      0 |      0 |
+------+--------+--------+
```
### Expected
```sql
+------+--------+--------+
| grp  | min(a) | max(a) |
+------+--------+--------+
| NULL |   NULL |   NULL |
+------+--------+--------+
```

**Comment by midenok** [comment]
Affected by `set_notnull()` in a1d5f71b0978127647921ed87626562c7c970a9d (#178).

**Comment by midenok** [comment]
Fixed in 3871121768294a12ace32afe16a3d07bce5e47a6 (#178, rebase).
